README.md

```diff
@@ -1,2 +1,31 @@
-# third-party-test-harness
+# Third-Party-Test-Harness
+
+## Description:
 A small Java web application that is used primarily for the use of karate testing and interacting with CHS services. The testing with this harness focuses on obtaining the correct scopes and permissions that are requested via this web application for use of Companies House Enabled API's
+This is an example application of a third party app gaining an Oauth token and using it for access and displaying,
+the correct scopes for access to API's. 
+
+### Requirements:
+- [Java](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+- [MongoDB](https://www.mongodb.com)
+
+### Usage:
+* In application.properties file changes need to be made to environment variables in order to get the application to work
```

Comment:
We are setting up the environment variables for CHS environment but they can change the values either in application.properties or set up env variables themselves

Reply:
We have the configuration section which explains how to get this working with the environment variables. If we want to explain that they can change the properties file instead, I would add more details rather than making it look like you need to amend it?